examples/light-switch-app/efr32/src/AppTask.cpp

```diff
@@ -157,15 +157,15 @@ CHIP_ERROR AppTask::Init()
     err = BaseApplication::Init(&gIdentify);
     if (err != CHIP_NO_ERROR)
     {
-        EFR32_LOG("BaseApplication::Init() failed");
+        SILABS_LOG("BaseApplication::Init() failed");
```

Comment:
SL_LOG would be shorter

Reply:
Thanks for the suggestion, will check in the next PR about this change